### ISSUE TITLE
Ensure category app settings are not included in remote project sync

### ISSUE THREAD

**Issue by mikkonie** [bihealth/sodar-core#1160]
Related to #20, #1151 and #1159.

I need to ensure app settings are not include in remote project sync as data provided to the target site. This may already be the case but tbh I can't remember right now.

The implementation of remote project sync changes in #638, so this has to be done either within that task or after it has been merged into dev.

**Comment by mikkonie** [comment]
After #1169, this is handled in the source site side. Closing.
